src/beginners/main.kt

```diff
@@ -2,5 +2,5 @@ package beginners
 
 fun main() {
         println("Hello World!")
-        println("How are you?")
+       // println("How are you?")
```

Comment:
Instead of commenting this line, I would recommend you to delete this line. And moreover this is a best practice.